website/content/docs/concepts/iam.mdx

```diff
@@ -0,0 +1,103 @@
+
+#### Scopes
+[Scopes](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/scopes) are containers for all of Boundary's resources and have their own permissions 
+boundary, isolated from other scopes at the same level. **Each scope has its own set of resources and permissions**. There are three types of scopes:
+- **Global** - The single, outermost scope of Boundary that cannot be deleted.
+- **Org (Organization)** - Org scopes are contained in the global scope and are typically used to represent business units, organizations, or departments. Org scopes 
+are also utilized when there are multiple production-level auth methods that need to be kept in separate scopes.
+- **Project** - Project scopes are contained in Org scopes and contain infrastructure resources.
+
+#### Auth Methods
+[Auth Methods](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/auth-methods) are resources that provide a mechanism for users to authenticate to 
+Boundary. Auth methods can only be contained by the global or Org scope. The available auth methods are:
+- Username/Password
+- [OIDC](https://developer.hashicorp.com/boundary/tutorials/access-management/oidc-auth)
+
+#### Accounts
+[Accounts](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/accounts) are resources that represent a set of credentials used to establish the 
+identity of a user. An account is created with an auth method, and can be associated with a single user. Accounts can be contained by any scope.
+
+#### Users
+[Users](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/users) are resources that represent an individual person or entity for the purposes of 
+access control. A user can contain multiple accounts from multiple auth methods, and can be contained by any scope.
+
+#### Groups
+[Groups](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/groups) are resources that represent a collection of users, and can be contained by any 
+scope. [Managed groups](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/managed-groups) are a special kind of group that syncs with the auth 
+method's identity provider to provide up to date information.
+
+#### Roles
+[Roles](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/roles) are resources that contain a collection of 
+[permissions](https://developer.hashicorp.com/boundary/docs/concepts/security/permissions) which are granted to any principal (user or group) from any scope. Roles 
+can be contained by any scope, and can specify which scope the permissions will apply to.
+
+## Access Management
+#### Configuring Users
+When adding new users to Boundary using the password auth method, there are 3 steps to follow. Note: for OIDC auth methods, the following will be auto-generated using 
```

Comment:
I might suggest breaking out OIDC/LDAP user and account configuration.  Also,
OIDC/LDAP users and accounts can be auto-vivified if they are the primary auth
method... and they can also be provisioned directly via the API.